### PR TITLE
Revamp of intro

### PR DESCRIPTION
**Pull request by jessitron**
Revamping the intro to newer messaging ... also Ryan said he wants the docs to have a voice, so this version is less neutral.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,154 +1,117 @@
 # Atomist - How Teams Deliver Software
 
+Custom software is crucial to competitive advantage in business. And advantage in custom software
+comes with a custom development experience, including delivery to a unique combination of production environments,
+in your ideal flow.
```
Comment (ddgenome) [inline docs/index.md]: This sounds like marketing-speak that would be more at home on the main website.

File: docs/index.md [hunk 2]
```diff
@@ -1,154 +1,117 @@
...
+
+Every serious business has its own development and delivery experience. Most of them
+aren't what we want. We want to spend our time and focus on delivering business value, but to do that well, 
+we need to enact our own blueprint for software delivery.
```
Comment (ddgenome) [inline docs/index.md]: Same. Maybe focus on teams and solving problems rather than business and value?

File: docs/index.md [hunk 3]
```diff
@@ -1,154 +1,117 @@
...
+
+So far, enterprises piece together pipeline tools, configured by a plethora of YAML or by hand in a GUI. For
+anything interesting, we resort to a Bash script, the lowest common denominator of programming.
```
Comment (ddgenome) [inline docs/index.md]: Ouch

File: docs/index.md [hunk 4]
```diff
@@ -1,154 +1,117 @@
...
+comes with a custom development experience, including delivery to a unique combination of production environments,
+in your ideal flow.
+
+Every serious business has its own development and delivery experience. Most of them
+aren't what we want. We want to spend our time and focus on delivering business value, but to do that well, 
+we need to enact our own blueprint for software delivery.
+
+So far, enterprises piece together pipeline tools, configured by a plethora of YAML or by hand in a GUI. For
+anything interesting, we resort to a Bash script, the lowest common denominator of programming. 
+We enroll myriad chatbots to spam our channels. We write how-to Wiki pages and distribute hand-me-down scripts.
```
Comment (ddgenome) [inline docs/index.md]: Typically wiki is not considered a proper noun.

File: docs/index.md [hunk 5]
```diff
@@ -1,154 +1,117 @@
...
+As our practices improve, older projects languish in prior standards. Meanwhile we shake our heads wistfully over 
+the elegant flow of shiny new delivery tools -- they will never fit our real-world environment.
```
Comment (ddgenome) [inline docs/index.md]: I like the message in this paragraph.

File: docs/index.md [hunk 6]
```diff
@@ -1,154 +1,117 @@
...
--   If you're new to Atomist, visit the [Atomist web site][www] to
-    learn more about Atomist and how we can help you and your team
-    deliver better software faster.
-
--   If you want to get started using Atomist, go to the [Using
+## What is this better way??
+
+Atomist lets you construct your delivery process in code -- but not too much code.
+A service, a framework, and some libraries take care of the pieces that are common to every
+development organization. It works atop your existing toolchain, adding functionality
```
Comment (ddgenome) [inline docs/index.md]: "It" is ambiguous here.

File: docs/index.md [hunk 7]
```diff
@@ -1,154 +1,117 @@
...
+## What is this better way??
+
+Atomist lets you construct your delivery process in code -- but not too much code.
+A service, a framework, and some libraries take care of the pieces that are common to every
+development organization. It works atop your existing toolchain, adding functionality 
+and smoothing your experience; then you're free to improve it.
+
+The crux of your development experience lives in your **software delivery machine** (SDM).
+This is a service that runs wherever you choose to run it. Start with one of ours,
+then make it yours. Your SDM is in TypeScript (or JavaScript works too), and comes with
```
Comment (ddgenome) [inline docs/index.md]: or any language that compiles to JavaScript?

File: docs/index.md [hunk 8]
```diff
@@ -1,154 +1,117 @@
...
+a framework designed for software delivery and development automation. Write functions to make decisions
+or take action, with access to all the code plus the context of the push or build or issue event.
+All of this is open source.
+
+While you can run an SDM independently and in private to help only yourself, the magic happens
+when it connects to the Atomist service to respond to your whole team or organization.
+The service provides triggering with rich events and custom commands; interactive chat integration;
```
Comment (ddgenome) [inline docs/index.md]: It seems regular commas would suffice here.

File: docs/index.md [hunk 9]
```diff
@@ -1,154 +1,117 @@
...
+
+I've talked about how your delivery flow can be defined in code instead of Bash, 
+in one open place instead of many narrow pipelines. I mentioned custom commands, where you can ensconce 
+common developer activities in a convenient location, accessible from chat. These commands are code:
+consistent, shared, versioned. You might have gleaned that because your Software Delivery Machine is code, you're
+never limited to plugins, nor to anyone else's idea of the correct delivery mechanism; your SDM can integrate with your existing tools. [Keep the tools that are working for you][favorite-tools-blog], and integrate new ones
+as you wish. Tie these together with an SDM, plus bonuses like [automatic CHANGELOG management][changelog-pack].
+
+Your SDM gives you something else: the ability to manipulate code across all your projects with one command. 
+Atomist libraries help you write code transforms -- functions that operate on code -- and the API for Software
```
Comment (ddgenome) [inline docs/index.md]: There should not be spaces surrounding an em dash.

File: docs/index.md [hunk 10]
```diff
@@ -1,154 +1,117 @@
...
+turns those into branches, pull requests, or automatic commits on every deviating push. Bring code up to standards
+and then keep it there.
+
+Your SDM gives you something else: start new projects with the right code and setup, every time. Atomist generators
+start from a real, working project and use code transforms to construct a starting point in a new repository. Set up
+any other 
+
+Your SDM can respond to [more events], and to custom events that you send. You can query your correlated events
```
Comment (ddgenome) [inline docs/index.md]: Should `[more events]` have a link?

File: docs/index.md [hunk 11]
```diff
@@ -1,154 +1,117 @@
...
+[spring-one]: https://www.youtube.com/watch?v=VDCHnTPknsI (Rod Johnson speaks at Spring One 2018)
+[spring-boot-to-k8s-blog]: https://the-composition.com/deploy-your-spring-boot-application-to-kubernetes-in-3-mins-fdd37a212c6c (Christian Dupuis's Spring Boot to Kubernetes Blog)
+[autofix-blog-test]: https://the-composition.com/making-change-stick-with-code-transforms-and-autofixes-587d19e0ba1b (Rod blogs about upgrading test file names)
+[autofix-blog-stars]: https://the-composition.com/align-the-stars-programmatically-35dc5625f97d (Jess blogs about autofixing comment formatting)
+[schema-deploy-blog]: https://the-composition.com/automation-story-graphql-schema-deployment-7893eb55ed18 (Jess blogs about graphql schema deploy)
+
+## You might want to know
+
+*  [Existing integrations][integrations]
+*  [Architecture][]
```
Comment (ddgenome) [inline docs/index.md]: Will that work if the reference is all lower case?

File: docs/integrations.md [hunk 12]
```diff
@@ -0,0 +1,16 @@
+Atomist receives events from and performs operations on many systems,
+including version control, CI systems, Slack, and more.
```
Comment (ddgenome) [inline docs/integrations.md]: If CI has systems, why wouldn't version control? I'd recommend being consistent.

File: docs/integrations.md [hunk 13]
```diff
@@ -0,0 +1,16 @@
...
+
+Atomist uses the native integration technology for each platform or
+tool.  For example, to integrate with GitHub and Travis CI, Atomist
+uses webhooks; to integrate with Slack it uses their native real-time
+messaging (RTM) API.  For each platform Atomist integrates with, it
```
Comment (ddgenome) [inline docs/integrations.md]: We no longer use RTM, we use their event API, which I think is also webhook driven.

File: mkdocs.yml [hunk 14]
```diff
@@ -67,6 +67,7 @@ pages:
       - Built-in Chat Integrations: lifecycle.md
       - Spring: user/spring.md
       - Kubernetes: user/kubernetes.md
+      - Supported Integrations: integrations.md
```
Comment (ddgenome) [inline mkdocs.yml]: Should this be under user/ so the URLs are consistent?

File: docs/index.md (hunk 2 above)
Comment (jessitron) [reply docs/index.md]: but ... but ... I want to deliver business value

File: docs/index.md (hunk 11 above)
Comment (jessitron) [reply docs/index.md]: yes, it is not case-sensitive

File: docs/integrations.md (hunk 12 above)
Comment (jessitron) [reply docs/integrations.md]: I cut & pasted this from the original intro page

File: docs/integrations.md (hunk 13 above)
Comment (jessitron) [reply docs/integrations.md]: great!

File: mkdocs.yml (hunk 14 above)
Comment (jessitron) [reply mkdocs.yml]: yes, great point.

**Pull request merged**